server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -78,6 +80,7 @@
     private GracefulShutdownTimeout gracefulShutdownTimeout;
     private ReplicationConfig replicationConfig = ReplicationConfig.NONE;
     private Ini securityConfig;
+    private Set<String> administrators;
```

Comment:
I'm curious if we should make this `final` and make `administrators()` always add, instead of setting:

```java
// foo and bar are both admins.
builder.admin('foo');
builder.admin('bar');
```